test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,30 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Server;
+import com.googlecode.jmxtrans.model.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
jmxtrans-agent has very strong requirements to not have any external dependencies. So unless you reimplement the AWS API yourself, there is not much chance that the PR would be approved. But it would be a welcomed addition to jmxtrans2 as an [additional writer](https://github.com/jmxtrans/jmxtrans2/tree/master/jmxtrans2-additional-writers).